### ISSUE TITLE
authenticateWithToken blocked?

### ISSUE THREAD

**Issue by AlizerUncaged** [realcoloride/node_characterai#21]
I have this code
```
async function main() {
    /// character.ai 
    await characterAI.authenticateWithToken("[token from local storage]");
    const characterId = "g60H0IyiUL2l4IvjRzDb7P5PyzY-H1auahH_kyfnWCk" 
    const chat = await characterAI.createOrContinueChat(characterId);
}
```
And it causes
```
Unhandled error occured but ignored. SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at parseJSONFromBytes (node:internal/deps/undici/undici:6498:19)
    at successSteps (node:internal/deps/undici/undici:6472:27)
    at node:internal/deps/undici/undici:1145:60
    at node:internal/process/task_queues:140:7
    at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
    at AsyncResource.runMicrotask (node:internal/process/task_queues:137:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

I have also checked that it seems like Cloudflare is blocking the request, looks like this is gonna be hard.

**Comment by realcoloride** [comment]
See #17

**Comment by realcoloride** [comment]
Fixed in #17
